src/BasePlatform.js

```diff
@@ -113,4 +113,29 @@ export default class BasePlatform {
     reload() {
         throw new Error("reload not implemented!");
     }
+
+    supportsAutoLaunch(): boolean {
```

Comment:
we don't have type annotations anywhere else. We probably should have them, but for now I'd say don't include them as it introduces confusion.

Reply:
nvm, it is worse than that: we do use it in the scope of electron. I'm not going to block this PR on that :/